www/src/pages/en/introduction.md

```diff
@@ -33,13 +33,18 @@ The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t
 
```

Comment:
Line 32:

What's `create-t3-app` then? Some kinda template?

Coming from the section above, "This" points towards the T3 Stack

Reply:
Changed the heading, here and in readme